src/alfasim_score/converter/alfacase/convert_alfacase.py

```diff
@@ -26,6 +30,27 @@ def _convert_well_trajectory(self) -> ProfileDescription:
         x, y = self.score_input.read_well_trajectory()
         return ProfileDescription(x_and_y=XAndYDescription(x=x, y=y))
 
+    def convert_materials(self) -> List[MaterialDescription]:
+        """Convert list of materials from SCORE file"""
+        material_descriptions = []
+        material_list = (
+            [self.score_input.read_cement_material()]
```

Comment:
It seems a good idea to change `self.score_input.read_cement_material()`, even if it has a single entry.
If later it requires to include more cements, it will be easier to perform the changes.

Reply:
I agree. Changed.